unittest/bindings/test_costs.py

```diff
@@ -260,19 +260,19 @@ class FrameVelocityCostSumTest(CostModelSumTestCase):
                                              pinocchio.LOCAL))
 
 
-if __name__ == '__main__':
-    test_classes_to_run = [
-        StateCostTest, StateCostSumTest, ControlCostTest, ControlCostSumTest, CoMPositionCostTest,
-        CoMPositionCostSumTest, FramePlacementCostTest, FramePlacementCostSumTest, FrameTranslationCostTest,
-        FrameTranslationCostSumTest, FrameRotationCostTest, FrameRotationCostSumTest, FrameVelocityCostTest,
-        FrameVelocityCostSumTest
-    ]
-    loader = unittest.TestLoader()
-    suites_list = []
-    for test_class in test_classes_to_run:
-        suite = loader.loadTestsFromTestCase(test_class)
-        suites_list.append(suite)
-    big_suite = unittest.TestSuite(suites_list)
-    runner = unittest.TextTestRunner()
-    results = runner.run(big_suite)
-    sys.exit(not results.wasSuccessful())
+# test to be run
```

Comment:
What is the reason behind removing the `if __name__ == '__main__'` from these tests?

Reply:
I created cmake target commands to be able to run specific python unit tests. But if I run them with `if __name__ == '__main__'`, then they don't trigger failures in the tests.

Reply:
this is because you are not running the python files, but importing them.

Reply:
@nim65s:

Is bad to remove the `if __name__ ...`?
If so, could you help me with a working solution?

Reply:
It would be better to keep it if we can make the test-targets work